.travis/platform.sh

```diff
@@ -0,0 +1,15 @@
+if [ -z "${PLATFORM:-}" ]; then
+  PLATFORM=$TRAVIS_OS_NAME;
```

Comment:
Inconsistent indentation.

Reply:
I imported that file unmodified from that Lua travis repo

Reply:
Point taken.

Reply:
I did modify setup_lua.sh though to cut it down to just building and installing Lua rocks

Reply:
If you didn't modify this file at all, I think it's OK to leave the indentation as-is.